lib/modules/manager/github-actions/extract.ts

```diff
@@ -7,7 +7,7 @@ import type { PackageDependency, PackageFile } from '../types';
 
 const dockerRe = regEx(/^\s+uses: docker:\/\/([^"]+)\s*$/);
 const actionRe = regEx(
-  /^\s+-?\s+?uses: (?<replaceString>['"]?(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>.+?)\s*['"]?(?:#\s+(?:renovate:\s+)?tag=(?<tag>.+?))?)\s*?$/
+  /^[^\S\r\n]+-?\s+?uses: (:?(?<replaceString>['"]?(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>[\w.]+)?[^\S\r\n]*['"]?[^\S\r\n]*(:?#\s*(?:renovate:\s+)?tag=(?<tag>[v?\d+.]+))?)(:?.*)?)$/
```

Comment:
There are never newlines, as we split the file and work line-by-line, see L19

Reply:
thanks for the heads up, 
those are left overs from working on a regex sim and multiline input.
fixed.